src/libsrx/src/common.h

```diff
@@ -26,6 +26,7 @@ int asprintf(char **strp, const char *fmt, ...);
 #define INFO(fmt, ...)  syslog(LOG_INFO, fmt, ##__VA_ARGS__)
 #define NOTE(fmt, ...)  syslog(LOG_NOTICE, fmt, ##__VA_ARGS__)
 #define WARN(fmt, ...)  syslog(LOG_WARNING, fmt, ##__VA_ARGS__)
+#define PANIC(fmt, ...) syslog(LOG_EMERG, fmt, ##__VA_ARGS__)
```

Comment:
To me, "panic" suggests that the system will be halted/rebooted or similar, not just that a log message is emitted.

In keeping with the other log macros, maybe we should stick to the syslog level names? I.e. change this macro to `EMERG()`?

Reply:
You caught that green goblin, like catnip to you, those goblins :smirk_cat: 

Will fix, was debating with myself long and hard between `EMERG()` and `PANIC()`, but I agree with you.  We should save the latter for cases when we need to actually panic (and possibly reboot using `watchdogd`).